libqtile/widget/__init__.py

```diff
@@ -32,17 +32,19 @@
 
 
 def safe_import(module_name, class_name):
-    safe_import_((".widget", module_name), class_name, globals(),
-                 fallback=make_error)
+    safe_import_(
+        (".widget", module_name), class_name, globals(), fallback=make_error)
 
 
 safe_import("backlight", "Backlight")
 safe_import("battery", ["Battery", "BatteryIcon"])
 safe_import("currentlayout", ["CurrentLayout", "CurrentLayoutIcon"])
 safe_import("currentscreen", "CurrentScreen")
 safe_import("debuginfo", "DebugInfo")
-safe_import("graph", ["CPUGraph", "MemoryGraph", "SwapGraph", "NetGraph",
-                      "HDDGraph", "HDDBusyGraph"])
+safe_import("graph", [
+    "CPUGraph", "MemoryGraph", "SwapGraph", "NetGraph", "HDDGraph",
+    "HDDBusyGraph"
+])
```

Comment:
Please keep the diff related to your feature.

Reply:
This was my code formatter, I think. I didn't do it on purpose.
I will revert it. Sorry for the inconvenience!.